packages/services/src/kernel/messages.ts

```diff
@@ -1030,6 +1030,13 @@ export interface IExecuteRequestMsg extends IShellMessage<'execute_request'> {
   };
 }
 
+/**
+ * Test whether a kernel message is an `'execute_request'` message.
+ */
+export function isExecuteRequestMsg(msg: IMessage): msg is IExecuteRequestMsg {
```

Comment:
Looking at the current diff, is this function still needed?

Reply:
Good point - not needed, but still useful and consistent with similar functions elsewhere. I was surprised it wasn't defined for consistency. If you want, I can delete it.

Reply:
Deleted...